_maps/map_files/generic/CentCom.dmm

```diff
@@ -2320,6 +2322,7 @@
 /area/ctf)
 "ha" = (
 /obj/machinery/power/emitter/energycannon{
```

Comment:
Is this subtype of emitter used anywhere other than CTF these days?

Reply:
often used for badminry as well